cmd/root_test.go

```diff
@@ -0,0 +1,212 @@
+/*
+This file is part of REANA.
+Copyright (C) 2022 CERN.
+
+REANA is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+package cmd
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"reanahub/reana-client-go/utils"
+	"reanahub/reana-client-go/validation"
+	"strings"
+	"testing"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/spf13/viper"
+)
+
+func testCmdRun(
+	t *testing.T,
+	cmd, serverPath, serverResponse string,
+	wantError bool, expectedMsgs []string,
+	args ...string,
+) {
```

Comment:
This function has quite a lot of parameters and it currently only support one endpoint, but we can improve this later when needed!